build-farm/platform-specific-configurations/aix.sh

```diff
@@ -76,7 +76,10 @@ fi
 
 
 if [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]; then
-  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --disable-warnings-as-errors --with-openssl=fetched"
+  if [ "$JAVA_FEATURE_VERSION" -ge 11 ]; then
+    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --disable-warnings-as-errors --with-openssl=fetched"
+  else
+    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched"
```

Comment:
You'll probably want the closing `fi` here.

Reply:
https://github.com/AdoptOpenJDK/openjdk-build/pull/1524

Reply:
Whoops! Thanks for spotting that silly mistake of mine :)